.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           command: bundle exec rspec spec
       - run:
           name: check old URLs
-          command: bundle exec ./scripts/sitemap-check --directory _site --old-urls-file OLD_URLS.yml
+          command: make test-urls
```

Comment:
This will trigger a second (unnecessary) build of the site. I'm assuming this is why the previous "run tests" build step `bundle exec rspec spec` duplicates the contents of `make test` rather than call it directly.

Reply:
Yeah good point, I'll just have this do that